src/storage/transaction/ChainResumeProcessor.cpp

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "storage/transaction/ChainResumeProcessor.h"
+#include "storage/transaction/ChainUpdateEdgeProcessorLocal.h"
+#include "storage/transaction/ChainAddEdgesProcessorLocal.h"
+#include "storage/transaction/ConsistUtil.h"
+#include "storage/transaction/TransactionManager.h"
+
+namespace nebula {
+namespace storage {
+
+void ChainResumeProcessor::process() {
+    std::unordered_map<GraphSpaceID, std::vector<meta::cpp2::LeaderInfo>> leaders;
+    if (env_->kvstore_->allLeader(leaders) == 0) {
+        LOG(INFO) << "no leader found, skip any resume process";
+        return;
+    }
+    std::unique_ptr<kvstore::KVIterator> iter;
+    for (auto& leader : leaders) {
+        auto spaceId = leader.first;
+        LOG(INFO) << "leader.second.size() = " << leader.second.size();
+        for (auto& partInfo : leader.second) {
+            auto partId = partInfo.get_part_id();
+            auto prefix = ConsistUtil::primePrefix(partId);
```

Comment:
How about add a timestamp after partId for primePrefix? Because in worst case for every insert/update, the resueme will be triggered. (doublePrimePrefix is ok because it is not written in normal insert/update path)